lighthouse-core/lib/page-functions.js

```diff
@@ -316,28 +322,28 @@ function getNodePath(node) {
  *  - nodePath: 0,HTML,1,BODY,1,DIV,a,#document-fragment,0,SECTION,0,IMG
  *  - nodeSelector: section > img
  */
-function getNodeSelector(node) {
+function getNodeSelector(element) {
```

Comment:
we require this to be an `Element` (in the existing type, but also because we access `classList` and `parentElement`), so changed the param name, but it seemed like too much to rename the function to `getElementSelector`...